src/util/metrics/threads_linux.rs

```diff
@@ -92,18 +94,12 @@ impl Collector for ThreadsCollector {
 
         let tids = get_thread_ids(self.pid).unwrap();
         for tid in tids {
-            if let Ok(Stat {
-                name,
-                state,
-                utime,
-                stime,
-            }) = Stat::collect(self.pid, tid)
-            {
+            if let Ok(stat) = pid::stat_task(self.pid, tid) {
+                // Threads CPU time.
+                let total = cpu_total(&stat);
+                let name = stat.command;
```

Comment:
Let's combine this line with the next line:

```
let name = sanitize_thread_name(tid, &stat.command);
```

Reply:
Done.